crates/protocol/src/compression/ratio.rs

```diff
@@ -4,6 +4,77 @@
 //!
 //! [rc]: https://github.com/ethereum-optimism/optimism/blob/develop/op-batcher/compressor/ratio_compressor.go#L7
```

Comment:
you should use a permalink here to a particular commit, otherwise it will go stale